src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/CourtSentencingNomisApiMockServer.kt

```diff
@@ -57,4 +58,5 @@ class CourtSentencingNomisApiMockServer(private val objectMapper: ObjectMapper)
     )
   }
   fun verify(pattern: RequestPatternBuilder) = nomisApi.verify(pattern)
+  fun verify(count: Int, pattern: RequestPatternBuilder) = MappingApiExtension.mappingApi.verify(count, pattern)
```

Comment:
not sure this method should be here, or if it is worth having 🤔

Reply:
yes- looks like I've mixed up my mocks - thanks!